modules/op-entitlements-shared-csi-driver.adoc

```diff
@@ -0,0 +1,123 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/configuring-security-context-for-pods.adoc
+:_mod-docs-content-type: PROCEDURE
+[id="op-entitlements-shared-csi-driver_{context}"]
+= Using Red Hat entitlements by sharing the secret using the Shared Resources CSI driver operator
+
+You can set up sharing of the `etc-pki-entitlement` secret from the `openshift-config-managed` namespace to other namespaces using the Shared Resources Container Storage Interface (CSI) Driver operator. You can then configure your pipeline to use this secret for the Buildah task.
+
+.Prerequisites
+
+* You are logged on to your {OCP} cluster using the `oc` command line utility as a user with cluster administrator permissions.
+* You enabled the Shared Resources CSI Driver operator on your {OCP} cluster.
+
+.Procedure
+
+. Create a `SharedSecret` custom resource (CR) for sharing the `etc-pki-entitlement` secret by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f - <<EOF
+apiVersion: sharedresource.openshift.io/v1alpha1
+kind: SharedSecret
+metadata:
+  name: shared-rhel-entitlement
+spec:
+  secretRef:
+    name: etc-pki-entitlement
+    namespace: openshift-config-managed
+EOF
+----
+
+. Create an RBAC role that permits access to the shared secret by running the following command:
++
+[source,terminal]
+----
+$ oc apply -f - <<EOF
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: shared-resource-rhel-entitlement
+  namespace: pipeline-namespace # <1>
+rules:
+  - apiGroups:
+      - sharedresource.openshift.io
+    resources:
+      - sharedsecrets
+    resourceNames:
+      - shared-rhel-entitlement
+    verbs:
+      - use
+EOF
+----
+<1> Replace `pipeline-namespace` with the namespace of your pipeline.
+
+. Assign the role to the `pipeline` service account by running the following command:
++
+[source,terminal]
+----
+$ oc create rolebinding shared-resource-rhel-entitlement --role=shared-shared-resource-rhel-entitlement \
+  --serviceaccount=pipeline
```

Comment:
Add namespace along with service account name





```suggestion
$ oc create rolebinding shared-resource-rhel-entitlement --role=shared-shared-resource-rhel-entitlement \
  --serviceaccount=<namespace>:pipeline
```
namespace is where your pipelinerun is running


Please rephrase with proper word :smile_cat: